bin/main.ml

```diff
@@ -2,6 +2,13 @@ open Base
 open Stdio
 open Nuscrlib
 
+let version_string () = "%%VERSION%%"
+
+let usage () =
+  "usage: "
+  ^ (Sys.get_argv ()).(0)
+  ^ " [-enum][-verbose][-fsm Role@Protocol][-project Role@Protocol] file"
```

Comment:
We should probably add other usages in another commit

Reply:
this is another issue